plugins/trendmicro_visionone/plugin.spec.yaml

```diff
@@ -93,35 +161,39 @@ types:
     last_modified_date_time:
       title: Last Modified Date Time
       type: string
-      description: Timestamp in ISO 8601 format that indicates the last time the information about a suspicious object was modified
+      description: Timestamp in ISO 8601 format that indicates the last time the information
+        about a suspicious object was modified
       required: true
       example: 2023-04-14T06:55:29Z
     scan_action:
       title: Scan Action
       type: string
-      description: Action that connected products apply after detecting a suspicious object
+      description: Action that connected products apply after detecting a suspicious
+        object
       required: true
       enum:
-        - block
-        - log
+      - block
+      - log
     risk_level:
       title: Risk Level
       type: string
       description: Risk level of a suspicious object
       required: true
       enum:
-        - high
-        - medium
-        - low
+      - high
+      - medium
+      - low
     in_exception_list:
       title: In Exception List
       type: boolean
-      description: Value that indicates if a suspicious object is in the exception list
+      description: Value that indicates if a suspicious object is in the exception
+        list
       required: true
     expired_date_time:
       title: Expired Date Time
       type: string
-      description: Timestamp in ISO 8601 format that indicates when the suspicious object expires
+      description: Timestamp in ISO 8601 format that indicates when the suspicious
+        object expires
```

Comment:
I think these double lined descriptions in your spec file are causing it to be improperly formatted which is what is giving you your refresh errors. Keep them all to one line regardless of how long they are and it should work better 😄